solutions/src/test/java/ru/sberbank/school/task13/beanfieldcopier/BeanFieldCopierTest.java

```diff
@@ -0,0 +1,27 @@
+package ru.sberbank.school.task13.beanfieldcopier;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import ru.sberbank.school.task13.beanfieldcopier.exception.BeanFieldCopierException;
+import ru.sberbank.school.task13.beanfieldcopier.model.IntData;
+import ru.sberbank.school.task13.beanfieldcopier.model.StringData;
+
+public class BeanFieldCopierTest {
+
+    @Test
+    public void copyTest() {
+         IntData from = new IntData(10);
```

Comment:
&#39;method def&#39; child has incorrect indentation level 9, expected level should be 8.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-79065631>*